external/buildscripts/build_runtime_vs.pl

```diff
@@ -27,20 +26,12 @@
 	'arch32=i'=>\$arch32,
 	'debug=i'=>\$debug,
 	'msbuildversion=s'=>\$msBuildVersion,
-	'il2cpp=i'=>\$il2cpp,
 	'gc=s'=>\$gc,
 ) or die ("illegal cmdline options");
 
 if ($build)
 {
-	if ($il2cpp)
-	{
-        CompileVCProj("$monoroot/msvc/mono-il2cpp.sln");
```

Comment:
We should remove mono-il2cpp.sln and the associated .targets files as well. That should be in a different PR. I can do it later.